spec/views/hq/orphans/show_spec.rb

```diff
@@ -11,7 +11,9 @@
       expect(rendered).to match @orphan.name
```

Comment:
My answer to this strangely brittle Minitest
```ruby
assert_select "input#partner_name" do
  assert_select "[value=?]", partner.name # with or without CGI::escape_html
end
```
is Capybara's clean, simple and (in my hands) robust
```ruby
expect(rendered).to have_field('Name', with: partner.name)
```